knexfile.js

```diff
@@ -1,18 +1,17 @@
-const localPgConnection = {
-  host: "localhost",
-  database: "rxid",
-  user: "admin",
-  password: "pass"
-};
+require("dotenv").config();
 
-const devDatabase = process.env.DATABASE_DEV || localPgConnection;
-const testDatabase = process.env.DATABASE_TEST || localPgConnection;
-const productionDatabase = process.env.DATABASE_URL || localPgConnection;
+const testDatabase = process.env.DATABASE_TEST;
+const productionDatabase = process.env.DATABASE_URL;
 
 module.exports = {
   development: {
     client: "pg",
-    connection: devDatabase + "?ssl=true",
+    connection: {
+      host: process.env.DEV_HOSTNAME || "localhost",
+      database: process.env.DEV_DATABASE || "rxid",
+      user: process.env.DEV_USER || "admin",
+      password: process.env.DEV_PASSWORD || "pass"
+    },
```

Comment:
Nice work getting the secrets into the environment variables. Make sure to include documentation in the readme where users can sign up for these, or pass them to the group. Feel free to set user name and empty pass to use a mandatory env var, instead of these defaults, when you use the production RDS database.